test/functional/api/cas/casadm.py

```diff
@@ -172,10 +172,11 @@ def stop_all_caches():
     if "No caches running" in list_caches().stdout:
         return
     TestRun.LOGGER.info("Stop all caches")
-    casctl_stop()
-    output = list_caches()
-    if "No caches running" not in output.stdout:
-        raise CmdException("Error while stopping caches.", output)
+    stop_output = casctl_stop()
+    caches_output = list_caches()
```

Comment:
We could also check exit code from those two and raise an exception accordingly,

Reply:
list_caches() checks its exit code